docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+```
+
+## Bring up the network with Certificate Authorities
+
+Hyperledger Fabric uses public key infrastructure (PKI) to verify the actions of
+all network participants. Every node, network administrator, and user submitting
+transactions need to have a public certificate and private key to verify their
+identity. These identities need to have a valid root of trust, establishing
+that the certificates and keys were issued by an organization that is a member
+of the network.
+
+The `network.sh` script must create all of the crypto material that is required
+to deploy and operate the network before it creates the peer and ordering nodes.
+
+By default, the script uses cryptogen to create the certificates and keys.
+cryptogen is a tool for development and testing that can quickly create the
+crypto material with a valid root of trust. When you run `network.sh up`, you
+can see the cryptogen tool creating the certificates and keys for Org1, Org2,
+and the Orderer Org.
+
+```
+creating Org1, Org2, and ordering service organization with crypto from 'cryptogen'
+
+/Usr/fabric-samples/first-network-new/../bin/cryptogen
+
+##########################################################
+##### Generate certificates using cryptogen tool #########
+##########################################################
+
+##########################################################
+############ Create Org1 Identities ######################
+##########################################################
++ cryptogen generate --config=./organizations/cryptogen/crypto-config-org1.yaml --output=organizations
+org1.example.com
++ res=0
++ set +x
+##########################################################
+############ Create Org2 Identities ######################
+##########################################################
++ cryptogen generate --config=./organizations/cryptogen/crypto-config-org2.yaml --output=organizations
+org2.example.com
++ res=0
++ set +x
+##########################################################
+############ Create Orderer Org Identities ###############
+##########################################################
++ cryptogen generate --config=./organizations/cryptogen/crypto-config-orderer.yaml --output=organizations
++ res=0
++ set +x
+```
+
+However, `network.sh` also provides the option to bring up the network using
+Certificate Authorities (CAs). In a production network, each organization would
+operate a CA (or multiple intermediate CAs) that creates the identities that
+belong to their organization. All the identities that are created by the organizations
+CA would share the same root of trust. Although it takes more time to run the
```

Comment:
All of the identities created by a CA run by the organization would share the same root of trust.

Reply:
done